.gitignore

```diff
@@ -1,14 +1,18 @@
 # directories
 artifacts
+artifacts-zk
 broadcast
 cache
+cache_hardhat-zk
```

Comment:
Can we use a hyphen here instead of an underscore?

Reply:
The folder, `cache_hardhat-zk`, is created by the hardhat so its not in our control.